test/testutil.c

```diff
@@ -185,18 +185,20 @@ static void test_fail_message(const char *prefix, const char *file, int line,
 static void test_fail_message_va(const char *prefix, const char *file, int line,
                                  const char *type, const char *fmt, va_list ap)
 {
-    fputs(prefix != NULL ? prefix : "ERROR", stderr);
-    fputs(":", stderr);
+    BIO *b_stderr = BIO_new_fp(stderr, BIO_NOCLOSE | BIO_FP_TEXT);
```

Comment:
That seems a bit silly to do *every* *time* there's a message to print, I'd rather the bigger harness opened the BIO and that `finalize()` closed it...